app/javascript/app/pages/login/login-actions.js

```diff
@@ -1,8 +1,9 @@
-import { PLANNING } from 'router';
+import { PLANNING, LOGIN } from 'router';
 
 // Action types
 const SET_USER = 'SET_USER';
 
 // Action creators
 export const setUser = data => ({ type: SET_USER, payload: data });
 export const navigateToPlanning = () => ({ type: PLANNING });
+export const navigateToLogin = () => ({ type: LOGIN });
```

Comment:
We can do the same using redux-tools following the same project structure.
For example: 
```
const navigateToLogin = createAction(LOGIN);
```